www/src/pages/en/usage/drizzle.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Drizzle
+description: Usage of Drizzle
+layout: ../../../layouts/docs.astro
+lang: en
+---
+
+Drizzle ORM is dialect specific, slim, performant and serverless ready by design.
+
+## Drizzle Client
+
+Located at `src/server/db/index.ts`, the Drizzle Client exported to be used in your API routes. Unlike prisma, the Drizzle client is not included in the [Context](/en/usage/trpc#-serverapitrpcts).
+
+## Schema
+
+You will find the Drizzle schema file at `src/server/db/schema.ts`. This file is where you define your database schema and models, but you can separate it into multiple files if you want, see how to do it at the [Drizzle schema documentation](https://orm.drizzle.team/docs/sql-schema-declaration). Drizzle is dialect specific and currently supports MySQL, PostgreSQL and SQLite.
+
+### With NextAuth.js
+
+When you select NextAuth.js in combination with Drizzle, the schema file is generated and set up for you with the recommended values for the `User`, `Session`, `Account`, and `VerificationToken` models, as per the [NextAuth.js documentation](https://authjs.dev/reference/adapter/drizzle).
+
+## Default Database
+
+The default database is an MySql database, using the PlanetScale serverless client, so it is edge ready out of the box! All you need to do is spin up a database at PlanetScale (up to one instance in free-tier BTW) and copy and paste the connection string in the `.env` file. Don't forget to add `ssl={"rejectUnauthorized":true}` to the end of the string.
```

Comment:
The default database is a* MySql database

Reply:
Oops.